api/tests.py

```diff
@@ -71,3 +80,274 @@ def test_options_manifest_data(self):
+    def setUp(self):
+
+        self.test_document_json = """
+            {
+            "title": "Test Reference Document",
+            "slug": "test-doc",
+            "desc": "This is a test document",
+            "license": "Open Gaming License",
+            "author": "John Doe",
+            "organization": "Open5e Test Org",
+            "version": "9.9",
+            "copyright": "",
+            "url": "http://example.com"
+            }
+        """
+
+        self.test_monster_json = """
+        {
+            "name": "Goblin",
+            "size": "Small",
+            "type": "humanoid",
+            "group": "Bad Guys",
+            "subtype": "goblinoid",
+            "alignment": "neutral evil",
+            "armor_class": 15,
+            "hit_points": 7,
+            "hit_dice": "2d6",
+            "speed": "30 ft.",
+            "strength": 8,
+            "dexterity": 14,
+            "constitution": 10,
+            "intelligence": 10,
+            "wisdom": 8,
+            "charisma": 8,
+            "stealth": 6,
+            "damage_vulnerabilities": "",
+            "damage_resistances": "",
+            "damage_immunities": "",
+            "condition_immunities": "",
+            "senses": "darkvision 60 ft., passive Perception 9",
+            "languages": "Common, Goblin",
+            "challenge_rating": "1/4",
+            "special_abilities": [
+                {
+                    "name": "Nimble Escape",
+                    "desc": "The goblin can take the Disengage or Hide action as a bonus action on each of its turns.",
+                    "attack_bonus": 0
+                }
+            ],
+            "actions": [
+                {
+                    "name": "Scimitar",
+                    "desc": "Melee Weapon Attack: +4 to hit, reach 5 ft., one target. Hit: 5 (1d6 + 2) slashing damage.",
+                    "attack_bonus": 4,
+                    "damage_dice": "1d6",
+                    "damage_bonus": 2
+                },
+                {
+                    "name": "Shortbow",
+                    "desc": "Ranged Weapon Attack: +4 to hit, range 80/320 ft., one target. Hit: 5 (1d6 + 2) piercing damage.",
+                    "attack_bonus": 4,
+                    "damage_dice": "1d6",
+                    "damage_bonus": 2
+                }
+            ],
+            "speed_json": {
+                "walk": 30
+            },
+            "armor_desc": "leather armor, shield",
+            "page_no": 315
+        }
+        """
+        i = Importer(ImportOptions(update=True, append=False, testrun=False))
+        i.import_document(json.loads(self.test_document_json),ImportSpec("test_filename","test_model_class","import_monster"))
+        i.import_monster(json.loads(self.test_monster_json),ImportSpec("test_filename","test_model_class","import_monster"))
+
+    def test_get_monsters(self):
+        response = self.client.get(f'/monsters/?format=json')
```

Comment:
another style nit that doesn't matter: This doesn't need to be an f-string. I'd recommend just using regular strings unless an f-string is explicitly required.